README.md

```diff
@@ -3,29 +3,34 @@ Kivy
 
 <img align="right" height="256" src="https://raw.githubusercontent.com/kivy/kivy/master/kivy/data/logo/kivy-icon-256.png"/>
 
-Innovative user interfaces made easy.
+[Kivy](https://kivy.org) is an open-source [Python](https://www.python.org/)
+framework for developing GUI apps that work cross-platform, including desktop, 
+mobile and embedded platforms.
 
-Kivy is an open source, cross-platform [Python](https://www.python.org)
-framework for the development of applications that make use of innovative,
-multi-touch user interfaces.
 The aim is to allow for quick and easy interaction design and rapid prototyping
-whilst making your code reusable and deployable.
+whilst making your code reusable and deployable: Innovative user interfaces made
+easy.
 
-Kivy is written in Python and [Cython](http://cython.org/), based on OpenGL ES
-2, supports various input devices and has an extensive widget library. With the
```

Comment:
I think we shouldn't delete the mention of OpenGL ES 2. This may be important for developers.